alpha/lib/model/DeliveryProfileLive.php

```diff
@@ -397,6 +401,46 @@ protected function getRenderer($flavors)
 		return $renderer;
 	}
 	
+	protected function retrieveStreamIdsByBitrate ($streams)
```

Comment:
This is a missleading name, I would say it's more of a "filterStreamIdsByBitrate" method since it definitely doesn't retrieve anything.